satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -619,4 +619,26 @@ update offer (
 	where offer.expires_at >= offer.created_at
 )
 
-delete offer ( where offer.id = ? )
\ No newline at end of file
+delete offer ( where offer.id = ? )
+
+//--- user credit table ---//
+
+model user_credit (
+	key		user_id
```

Comment:
The `user_id` won't be unique in this table, we probably need a `id` as the primary key that's just a `serial`. I have a PR up for the new schema https://github.com/storj/storj/pull/2046/files#diff-081def8b8f7fc45ec1f5c91b31f5d030